buildscripts/kokoro/macos.cfg

```diff
@@ -15,6 +15,7 @@ env_vars {
 # We always build mvn artifacts.
 action {
   define_artifacts {
+    regex: "github/grpc-java/**/build/reports/**"
```

Comment:
Let's add this in another PR for easier netty rollback - if we ever need it?

Reply:
Just noticed this. There are other changes in this PR that are also needed even if we revert the netty upgrade - like the change in netty/src/test/java/io/grpc/netty/NettyTestUtil.java . So I'll create the other PR if/when needed.